amlip_docs/rst/user_manual/nodes/inference.rst

```diff
@@ -0,0 +1,65 @@
+.. include:: ../../exports/alias.include
+
+.. _user_manual_nodes_inference:
+
+##############
+Inference Node
+##############
+
+This node waits for data serialized as :ref:`user_manual_datatype_inference`, and once received it calculate the inference whose output is the inference solution as :ref:`user_manual_datatype_inference_solution`.
+
+.. warning::
+
+    The use of a Inference node must be synchronous.
```

Comment:
reformulate this. It makes sense in edge but not here. Say something more likely "must be called once per inference that want to process"

Reply:
This is still not correct. This process "does not send" anything, it receives something.
*This means that once `process_inference` is called the thread must wait for the data to arrive, and answer it, before sending another data.*